int8 slave_id
int8 func_code
int32 write_addr
int32 read_addr
int8 write_num
int8 read_num
int32[32] data

